Describes 2.1.0 compaction changes in rel notes (#261)
apache/accumulo-website: _posts/release/2020-01-19-accumulo-2.1.0.md

   compactions it can starve quick compactions.  Now compactions with little
   data can be processed by dedicated thread pools.
 * Accumulo's default algorithm for selecting files to compact was modified to
   select the smallest set of files that meet the compaction ratio criteria
   instead of the largest set.  This change makes tablets more aggressive about
   reducing their number files while still doing logarithmic compaction work.
   This change also enables efficiently compacting new small files that arrive
   during a long running compaction. 
 * Having dedicated compaction threads pools for tables is now supported
   through configuration.  The default configuration for Accumulo sets up
   dedicated thread pools for compacting the Accumulo metadata table.
 * Merging minor compactions were dropped.  These were added to Accumulo to
   address the problem of new files arriving while a long running compaction
   was running.  Merging minor compactions could cause O(N^2) compaction work.
   The new compaction changes in this release can satisfy this use case while
   doing a logarithmic amount of work.

> Accumulo has a pluggable user interface called CompactionStrategy, it was
> deprecated in this release and replaced with new interfaces in the public API.
> The javadocs for CompactionStrategy have more detailed information about its
> deprecation.  CompactionStrategy transitively referenced many Accumulo types
> that are not public API.  Therefore it is extremely difficult to know if
> compactions strategies written for one release of Accumulo will work with a
> later release.  The new interfaces that replace CompactionStrategy only use
> public API types making it easier to write code that works across releases.

Reviewer: The last paragraph could be shorter. It could just say:
```suggestion
"The CompactionStrategy interface was deprecated in favor of new interfaces in the public 
API. The javadocs for CompactionStrategy have more detailed information about its deprecation.
```
If the javadoc already includes the deprecation justification, just include a link to the javadoc.

Author: Good idea, I will link to the javadoc.  One irksome thing is I can't test the link ATM

Reviewer: Right, I hear ya.  You could put a link but make the text say something obvious (like REPLACE ME WITH RELEASED JAVADOC) so we know to replace it at release.

Author: I updated the text, it is much shorter.  Actually I can test the link. I tried t locally and it works.  It links to the 2.0.1 version of compactions strategy javadoc ATM.  So I can not test the full experience of linking to the 2.1.0 javadocs, but I can test that the link works.
